pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctCountThetaSketchAggregationFunction.java

```diff
@@ -1024,7 +1024,8 @@ public Comparable extractFinalResult(List<ThetaSketchAccumulator> accumulators)
     int numAccumulators = accumulators.size();
     List<Sketch> mergedSketches = new ArrayList<>(numAccumulators);
 
-    for (ThetaSketchAccumulator accumulator : accumulators) {
+    for (Object accumulatorObject : accumulators) {
```

Comment:
Please make this change in `DistinctCountRawThetaSketchAggregationFunction.extractFinalResult()` to ensure backward compatibility.

Reply:
Updated.